app/mailers/views/hypertension_fast_track_pilot_mailer.erb.html

```diff
@@ -6,16 +6,29 @@
   <body>
     <p>
       <% if @submission.form_json.include? "VAMC_Hypertension_Rapid_Decision_Evidence.pdf" %>
-        A single issue hypertension claim for increase was just submitted on va.gov. 
-        A health summary PDF was generated and added to the claim’s documentation.
+        A single-issue hypertension claim for increase was submitted on va.gov.<br/>
+        A health summary PDF was generated and added to the claim's documentation.
       <% else %>
-        A single issue hypertension claim for increase was just submitted on va.gov. 
+        A single-issue hypertension claim for increase was submitted on va.gov.<br/>
         There was not sufficient data to generate a health summary PDF associated with this claim. 
       <% end %>
     </p>
-      Submission ID: <%= @submission.id %>
-      Benefit Claim Id: <%= @submission.submitted_claim_id %>
-      Time Submission Was Made: <%= @submission.created_at %>
+      <table border="1" cellspacing="1" cellpadding="5">
+        <thead>
+          <tr>
+            <td>Submission ID</td>
+            <td>Benefit Claim Id</td>
+            <td>Time Submission Was Made</td>
+          </tr>
+        </thead>
+        </tbody>
```

Comment:
I think this needs to be an opening <tbody> tag

Reply:
Nice catch! Fixed.